src/orders/components/OrderCustomerAddressesEditDialog/utils.ts

```diff
@@ -2,9 +2,26 @@ import {
   CustomerAddresses_user_addresses,
   CustomerAddresses_user_defaultShippingAddress
 } from "@saleor/customers/types/CustomerAddresses";
+import { AccountErrorFragment } from "@saleor/fragments/types/AccountErrorFragment";
+import { OrderErrorFragment } from "@saleor/fragments/types/OrderErrorFragment";
+import { FormChange } from "@saleor/hooks/useForm";
 import { flatten } from "@saleor/misc";
+import { AddressTypeEnum } from "@saleor/types/globalTypes";
 
 import { getById } from "../OrderReturnPage/utils";
+import {
+  OrderCustomerAddressesEditData,
+  OrderCustomerAddressesEditHandlers
+} from "./form";
+import { OrderCustomerAddressEditProps } from "./OrderCustomerAddressEdit";
+import { OrderCustomerSearchAddressState } from "./types";
+
+interface AddressEditCommonProps {
+  showCard: boolean;
+  loading: boolean;
+  countryChoices: Array<{ label: string; value: string }>;
```

Comment:
```suggestion
  countryChoices: Choice[];
```

Or something like this, it's located somewhere near `SingleAutocompleteSelectField`